pyriemann/classification.py

```diff
@@ -829,55 +828,37 @@ def _set_kernel(self):
 
 
 class MeanField(SpdClassifMixin, TransformerMixin, BaseEstimator):
-    """Classification by Riemannian Means Field Classifier
+    """Classification by Minimum Distance to Mean Field.
 
-    The Riemannian Means Field Classifier (MF) [1]_ is the second version,
-    which improves the performance of the Minimum Distance to Mean Field
-    (MDMF) [2]_ classifier. Classification is performed by defining a list
-    of power means for each class.
+    Classification by Minimum Distance to Mean Field, defining several
+    power means for each class.
+
+    This implementation covers the first version of the algorithm
+    introduced in the conference paper [1]_.
+
+    For the article version [2]_, please see [3]_.
```

Comment:
```suggestion
    The Mean Field estimates several power means for each class.
    Then, it can be used as a classifier, which computes the minimum distance
    to the mean field [1]_;
    or as a feature extractor, which must be pipelined with another classifier
    [2]_.
```